src/create-verify-archive.js

```diff
@@ -76,6 +77,49 @@ exports.createCommand = function*(argv) {
     var absOutDir = path.resolve(outDir);
 
     yield repoutil.forEachRepo(repos, function*(repo) {
+        if(isWin) {
+            var autoCRLF;
+            var eol;
+            var msg = '';
+
+            try {
+                autoCRLF = yield executil.execHelper(executil.ARGS('git config --get core.autocrlf'), true);
+            } catch(e) {
+                autoCRLF = '';
+            }
+
+            try {
+                eol = yield executil.execHelper(executil.ARGS('git config --get core.eol'), true);
+            } catch(e) {
+                eol = '';
+            }
+
+            if(autoCRLF !== 'false') {
+                msg = 'Warning: core.autocrlf is set to "' + autoCRLF + '".\n' + 
+                    'Set either "' + repo.repoName + '" or global core.autocrlf setting to "false" to avoid issues with executables on non-Windows OS.\n';
+            }
+
+            if(eol !== 'lf') {
```

Comment:
According to https://www.kernel.org/pub/software/scm/git/docs/git-config.html `core.eol=native` is also appropriate value on Mac and Linux.